packaging.make

```diff
@@ -0,0 +1,27 @@
+BUILD_CHANNEL?=local
+
+appimage: build
+	cd etc/packaging/appimages && BUILD_CHANNEL=${BUILD_CHANNEL} appimage-builder --recipe cartographer-module-`uname -m`.yml
+	if [ "${RELEASE_TYPE}" = "stable" ]; then \
+		cd etc/packaging/appimages; \
+		BUILD_CHANNEL=stable appimage-builder --recipe cartographer-module-`uname -m`.yml; \
```

Comment:
Will we end up building two identical appimages when the release type is "stable"?

Reply:
Almost identical. The "channel" itself is embedded into the image, and determines the auto-update URL. So it's necessary to build one for each channel so that they use different update URLs.